### ISSUE TITLE
Tarball 3.15.1 is not updated

### ISSUE THREAD

**Issue by giuliobenetti** [libfuse/libfuse#812]
While bumping version for Buildroot I've realized that version 3.15.1 tarball doesn't contain sources from tag 3.15.1 but from tag 3.15.0.
Checking git I've also found that while checking out master branch all the commits of 3.15.1 are shown, but if I check out tag fuse-3.15.1 all the commits are not there.

@Nikratio can you please take a look?

Thanks a lot

**Comment by thesamesam** [comment]
It also seems to be `tar.gz` now instead of the preferred `tar.xz`?

**Comment by Nikratio** [comment]
Can you elaborate on what you mean with "arball doesn't contain sources from tag 3.15.1 but from tag 3.15.0."? If I do a git checkout of the fuse-3.15.1 tag, then it matches what's in the tarball.

**Comment by giuliobenetti** [comment]
> Can you elaborate on what you mean with "arball doesn't contain sources from tag 3.15.1 but from tag 3.15.0."? If I do a git checkout of the fuse-3.15.1 tag, then it matches what's in the tarball.

If you try to :
```
$ git checkout master
$ git log --oneline 
```
you will see all the commits up to 3.15.1:
```
d888c30 (HEAD -> master, upstream/master) Use signify to sign releases.
a98add6 Merge tag 'fuse-3.15.1'
0b50e26 (tag: fuse-3.15.1) Released fuse-3.15.1
1cb6e17 (origin/master, origin/HEAD) Reduce default write size by half
b51f69f Add missing include.
51bc827 Make expire only function fail if no kernel support (#789)
7567268 Clarify behavior of fuse_session_exit().
010d820 Improve wording of user_allow_other usage instructions (#806)
b58a001 Wrapper around test applications for cross compiler environment in meson.build (#804)
6d08472 (tag: fuse-3.15.0) Released 3.15.0
```
But if you:
```
$ git checkout fuse-3.15.1
$ git log --oneline 
```
You will only see this:
```
0b50e26 (HEAD, tag: fuse-3.15.1) Released fuse-3.15.1
6d08472 (tag: fuse-3.15.0) Released 3.15.0
e0041ad Error handling for fusermount's commfd (#786)
```
So all commits between 3.15.0 and 3.15.1 are not there(I don't know the reason).
So tarball is created starting from tag 3.15.1 but the problem seems to be in git.
If you compare branch master code and tarball code you will see that all the commits up to 3.15.1 are not there.
For example this one:
b58a001

**Comment by Nikratio** [comment]
> If you try to :
> 
> ```
> $ git checkout master
> $ git log --oneline 
> ```
> 
> you will see all the commits up to 3.15.1:

No, what you see there is all commits that are in the master branch. That is not the same thing as what's in the 3.15.1 release. The 3.15.1 release is - by definition - what's in the fuse-3.15.1 tag or (equivalently) in the 3.15.1 release tarball.

I guess what you are trying to say is that you would have preferred that some of the changes that are currently in master would have made it into the 3.15.1 release. That's a valid opinion to have, but not a bug in libfuse :-).

**Comment by giuliobenetti** [comment]
> > If you try to :
> > ```
> > $ git checkout master
> > $ git log --oneline 
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > you will see all the commits up to 3.15.1:
> 
> No, what you see there is all commits that are in the master branch. That is not the same thing as what's in the 3.15.1 release. The 3.15.1 release is - by definition - what's in the fuse-3.15.1 tag or (equivalently) in the 3.15.1 release tarball.
> 
> I guess what you are trying to say is that you would have preferred that some of the changes that are currently in master would have made it into the 3.15.1 release. That's a valid opinion to have, but not a bug in libfuse :-).

Please take a look at tag fuse-3.15.1:
https://github.com/libfuse/libfuse/commits/fuse-3.15.1

There are no commits between commits:
`Released fuse-3.15.1` and `Released 3.15.0`

Am I missing something?

**Comment by giuliobenetti** [comment]
> Take a look at Changes.txt

Aaah, so the fuse_3.15.1 release is a single commit. My bad.
Looking at master branch, where the commits for next release comes before the Release 3.15.1 commit lead me to the wrong path. And indeed here there are the commits after the release:
https://github.com/libfuse/libfuse/compare/fuse-3.15.1...master

Sorry for the noise and thank you for answering me :-)
Look forward for 3.16 then.

Kind regards
